05-ignore.md

```diff
@@ -6,13 +6,25 @@ minutes: 5
 ---
 > ## Learning Objectives {.objectives}
 > 
-> *   Configure Git to ignore specific files,
->     and explain why it is sometimes useful to do so.
+> *   Explain why it is sometimes useful to have Git ignore specific files
+> *   Configure Git to do so
+>      
 
-What if we have files that we do not want Git to track for us,
-like backup files created by our editor
-or intermediate files created during data analysis.
-Let's create a few dummy files:
+### Why it is sometimes useful to *not* track files
```

Comment:
Our lesson template doesn't use subheadings inside a given topic, so it's better to remove this line.